docs/developers/contributing/ci-cd.md

```
@@ -86,21 +91,20 @@ services:
 
 ### Batching Features
 
-You can merge multiple features before releasing:
+You can merge multiple features before releasing. The release-please Release PR accumulates on every merge regardless of how previews are built:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)